nix/yazi.nix

```diff
@@ -24,8 +24,7 @@
 }:
 
 let
-  runtimePaths = with lib; [ ]
-    ++ optional withFile file
+  runtimePaths = with lib; optional withFile file
```

Comment:
you can also remove this `with lib;` by doing `inherit (lib) optional makeBinPath;`